File: 1-Review/C/review-2.c
#pragma warning(disable:4996)
#include <stdio.h>

void ABC(int arr[], int i);

int main() {

	int arr[10];

	for (int i = 0; i < 10; i++) 
		scanf("%d", &arr[i]);

	for (int i = 0; i < 9; i++)
		ABC(arr, i);

	for (int i = 0; i < 10; i++)
		printf(" %d", arr[i]);

	return 0;
}

void ABC(int arr[], int i) {
	int max = arr[i];
	int max_index = i;

	for (int j = i; j < 10; j++) {
		if (max < arr[j]) {
			max = arr[j];
			max_index = j;
		}
	}

	int tmp = arr[max_index];
	arr[max_index] = arr[i];
	arr[i] = tmp;
}

File: 1-Review/Python/review-2.py
def ABC(arr, i):
    max = arr[i]
    max_index = i

    for j in range(i, len(arr)):
        if(max < arr[j]):
            max = arr[j]
            max_index = j

    arr[max_index], arr[i] = arr[i], arr[max_index]

def main():
    arr = []
    arr = list(map(int, input().split()))
    
    for i in range(9):
        ABC(arr, i)

    print(" ".join(map(str, arr)))        

if __name__ == '__main__':
    main()